app/assets/javascripts/application.js

```diff
@@ -37,3 +28,18 @@ function stripeResponseHandler(status, response) {
     $form.get(0).submit();
   }
 };
+
+$(document).ready(function() {
+  $("#use_saved_card").on("click", toggleCreditCardFields);
+});
+
+function toggleCreditCardFields(){
+  if ($("#use_saved_card")[0].checked == true) {
```

Comment:
Use `===` here instead of `==`. Due to type coercion in JavaScript, non-boolean values can be compared to `true` using `==` and still evaluate to true. Best practice is to use `===` which checks that both values are indeed booleans.